variant/validators/insertion_base.py

```diff
@@ -0,0 +1,104 @@
+
+    def get_valid_invalid_results(self, classification_tokens, transcripts,
+                                  classification, results, gene_tokens) \
+            -> None:
+        """Add validation result objects to a list of results.
+
+        :param list classification_tokens: A list of Tokens
+        :param list transcripts: A list of transcript strings
+        :param Classification classification: A classification for a list of
+            tokens
+        :param list results: A list to store validation result objects
+        :param list gene_tokens: List of GeneMatchTokens
+        """
+        valid_alleles = list()
+        mane_transcripts_dict = dict()
+        for s in classification_tokens:
+            for t in transcripts:
+                errors = list()
+
+                allele, t, hgvs_expr, is_ensembl = \
+                    self.get_allele_with_context(classification, t, s, errors)
+
+                if hgvs_expr not in mane_transcripts_dict.keys():
+                    mane_transcripts_dict[hgvs_expr] = {
+                        'classification_token': s,
+                        'transcript_token': t,
+                        'nucleotide': is_ensembl
+                    }
+
+                if allele:
+                    self.check_pos_index(t, s, errors)
+
+                self.add_validation_result(
+                    allele, valid_alleles, results,
+                    classification, s, t, gene_tokens, errors
+                )
+
+        # Now add Mane transcripts to results
+        self.add_mane_transcript(classification, results, gene_tokens,
+                                 mane_transcripts_dict)
+
+    def get_hgvs_expr(self, classification, t, s, is_hgvs):
+        """Return a HGVS expression.
+
+        :param Classification classification: A classification for a list of
+            tokens
+        :param str t: Transcript retrieved from transcript mapping
+        :param Token s: The classification token
+        :param bool is_hgvs: Whether or not classification is HGVS token
+        :return: hgvs expression
+        """
+        if not is_hgvs:
+            prefix = f"{t}:{s.reference_sequence.lower()}."
+            position = f"{s.start_pos_flank}_{s.end_pos_flank}"
+            if s.inserted_sequence2 is not None:
+                inserted_sequence = \
+                    f"{s.inserted_sequence}_{s.inserted_sequence2}"
+            else:
+                inserted_sequence = f"{s.inserted_sequence}"
+
+            hgvs_expr = f"{prefix}{position}ins{inserted_sequence}"
+        else:
+            hgvs_token = [t for t in classification.all_tokens if
+                          isinstance(t, Token) and t.token_type
+                          in ['HGVS', 'ReferenceSequence']][0]
+            hgvs_expr = hgvs_token.input_string
+        return hgvs_expr
+
+    def concise_description(self, transcript, token) -> str:
+        """Return a description of the identified variant."""
```

Comment:
params/return value

Reply:
Updated docstrings in all validators for concise_description. Should probably go ahead and make another issue to go back and update docs because it's inconsistent throughout. Sometimes params/return value are added and sometimes they aren't.